tests/lib/tools/nested-state

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -e
+
+show_help() {
+    echo "usage: prepare-env"
+    echo "       cleanup-env"
+    echo "       install-local-as <snap-name> <dest-name> [OPTIONS]"
```

Comment:
This is a bit copy-pasty from the other tool :)

Reply:
updated